src/lib/TextExtraction.js

```diff
@@ -5,17 +5,22 @@ class TextExtraction {
    *                              other options than pattern would be added to the parsed content
    * @param {RegExp} patterns[].pattern - RegExp to be used for parsing
    */
-  constructor(text, patterns) {
+  constructor(text, patterns, propsToPass) {
```

Comment:
I would argue that the `TextExtractor` is not concerned about ensuring properties from a parent are provided to the children. The role of this is to specifically generate values that pertain to the result of parsing the input. 

The responsibility couples with the actual `ParsedText` component itself, specifically in where the child `Text` components are prepared https://github.com/taskrabbit/react-native-parsed-text/blob/v0.0.10/src/ParsedText.js#L56-L63.

This line should be updated to the following:

```diff
 return (
   <React.Text
     key={`parsedText-${index}`}
+    {...this.props.childrenProps}
     {...props}
   />
 )
```

A bit less round-about, and incurs only two lines of significant code:
- The additional `PropType` definition
- Applying the properties to the chilren